### DISCUSSION TITLE
httpcert error

### DISCUSSION THREAD

**Discussion by shelu16** [dstotijn/hetty#34] [Q&A] [answered]
root1@SBCMUMNB76:~/tools/alltest$ hetty
2020/10/08 18:11:59 [INFO] Running server on :8080 ...
2020/10/08 18:12:34 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:39 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:40 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:41 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:41 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:42 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:42 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:43 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:43 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:46 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:49 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:13:55 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:19:05 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate
2020/10/08 18:19:06 [ERROR] Securing client connection failed: handshake error: remote error: tls: bad certificate

there is cert error how to solve i am using binary

**Comment by RiRa12621** [comment]
Which build version of hetty are you running?

**Comment by dstotijn** [comment, marked as answer]
@shelu16, most likely you didn't install the CA certificate to your system/browser CA store. We just updated the [README](https://github.com/dstotijn/hetty/blob/master/README.md) with instructions on how to do that.

**Comment by shelu16** [comment]
thanks i found your post 
https://github.com/dstotijn/hetty/pull/21/files/bbd89abae5d759a854934b238582db48fc159018
now its working

**Comment by shelu16** [comment]
![Uploading hetty.JPG…]()
